marklogic-client-api/src/test/java/com/marklogic/client/impl/OpticPlanSearchOptionsTest.java

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.client.impl;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.expression.PlanBuilder;
+import com.marklogic.client.row.RowManager;
+import com.marklogic.client.row.RowRecord;
+import com.marklogic.client.row.RowSet;
+import com.marklogic.client.test.Common;
+import com.marklogic.client.test.junit5.RequiresML12;
+import com.marklogic.client.type.PlanSearchOptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.Iterator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(RequiresML12.class)
+class OpticPlanSearchOptionsTest {
+	private DatabaseClient client;
+	private RowManager rowManager;
+	private PlanBuilder op;
+
+	@BeforeEach
+	void beforeEach() {
+		client = Common.newClient();
+		rowManager = client.newRowManager();
+		op = rowManager.newPlanBuilder();
+	}
+
+	@Test
+	void bm25() {
+		PlanBuilderBaseImpl.PlanSearchOptionsImpl options =
+			new PlanBuilderBaseImpl.PlanSearchOptionsImpl(new PlanBuilderSubImpl(), null, PlanSearchOptions.ScoreMethod.ZERO, null);
+		PlanBuilder.ModifyPlan plan = op.fromSearch(op.cts.wordQuery("contents"), null, null, options).offsetLimit(0, 5);;
+		RowSet<RowRecord> rowSet = rowManager.resultRows(plan);
```

Comment:
Use `resultRows` in `AbstractOpticUpdateTest`, which also returns a far more useful List instead of a RowSet.